### ISSUE TITLE
CNN feature peeking

### ISSUE THREAD

**Issue by MatthewJA** [chengsoonong/crowdastro#197]
Currently the CNN is trained on the full dataset. This is probably not a good idea as it allows the CNN to see all the labels and data and thus the features may actually encode the problem. This should be minimised by using a separate test set (necessarily small due to the size of ATLAS).

**Comment by MatthewJA** [comment]
Problem solved! I put up an AWS spot instance, ran a CNN on a dedicated training set, and made sure to freeze those indices. They are WISE only, so I'll need to either repeat this or remap it for SWIRE.
